### ISSUE TITLE
[chromecast] playStream(null) throws NullPointerException

### ISSUE THREAD

**Issue by Tereius** [openhab/openhab-addons#6740]
The docs state:
`playStream(String url)`: plays an audio stream from an url to the default sink (set url to `null` if streaming should be stopped)

Following minimal rule will throw a NullPointerException and won't stop the stream (tested version 2.5.0):
```
import java.lang.Throwable

rule "test"
when
    Time cron "0/5 * * ? * * *"
then
    try {
        playStream(null)
    } catch(Throwable e) {
        logInfo("Test", e.getStackTrace().toString())
    }
end
```
The stack trace:
```
java.util.concurrent.ConcurrentHashMap.putVal(ConcurrentHashMap.java:1011)
java.util.concurrent.ConcurrentHashMap.put(ConcurrentHashMap.java:1006)
org.eclipse.smarthome.core.audio.internal.AudioServlet.serve(AudioServlet.java:171)
org.openhab.binding.chromecast.internal.ChromecastAudioSink.process(ChromecastAudioSink.java:56)
org.openhab.binding.chromecast.internal.handler.ChromecastHandler.process(ChromecastHandler.java:202)
org.eclipse.smarthome.core.audio.internal.AudioManagerImpl.play(AudioManagerImpl.java:143)
[...]
```

**Comment by cweitkamp** [comment]
Thanks for reporting. I submitted a fix for it (see  #6744).

**Comment by Tereius** [comment]
Thank you very much.
